text/0000-project-groups.md

```diff
@@ -0,0 +1,391 @@
+can be hard to know their current status.
+
+[ffi unwind]: https://github.com/rust-lang/project-ffi-unwind
+[inline asm]: https://github.com/rust-lang/project-inline-asm
+[safe transmute]: https://github.com/rust-lang/project-safe-transmute
+[triage]: https://github.com/rust-lang/team/blob/master/teams/wg-triage.toml
+[rustup]: https://github.com/rust-lang/team/blob/master/teams/wg-rustup.toml
+
+### Project Group Creation
+
+Since project groups are approved by their relevant parent team over the core
+team, it's up to each team decide their specific requirements. However we
+recommend using the [application checklist](#application-checklist) as the basis
+for process and if needed adding any extra requirements.
+
+Process around project group membership is up to the shepherd's discretion.
+Typically, people who are productively contributing to the project group for
+some time will be added as members. It is not required for a project group to
+have a lot of members though, some project groups may only have one or
+two members.
+
+### Project Group Evaluation
+
+Parent teams should add checking in with their project groups as part of their
+regular triage.
+
+## Community Groups
+
+Community Groups are groups of individuals who want to work and collaborate on a
+specific topic, without going through the formal process of a working group.
+Community groups are encouraged to adopt the structure of working or projects
+groups laid, but are free create and experiment with their own structure. As
+such community groups are not officially endorsed by The Rust Programming
+Language Organisation.
+
+## Archival
+
+The archival process of a group involves communicating the wind down to the
+community and the archival or transfer of ownership of the relevant projects.
+As well archiving any chat channels hosted by the Rust project.
+
+### Retrospectives
+
+While this RFC attempts to address some of the current organisational problems
+within the organisation, it also doesn't believe that this RFC will be a panacea
+to those problems or that we won't encounter more in the future. As part of
+that, we'd also like to introduce performing retrospectives with groups, once
+significant time has past or the group has been finished it's project.
+
+This would involve a discussion between the members of the group, and ideally
+their parent team and the Governance working group. The retrospective should
+produce a public blog post on the Inside Rust blog, however any feedback a
+member has that they would want to keep private would be omitted.
+
+The blog post should try to cover the output of the group, such as RFCs or
+projects, as well what the group thought worked and importantly what
+didn't work. This should help us iterate on this initial RFC and help us find
+and address issues that come up in the process.
+
+# Drawbacks
+
+- This will add organisational overhead to maintain group repositories
```

Comment:
I think the RFC as presented is pretty lightweight (a good thing!) and doesn't add too much overhead. I particularly like that it is a scalable process. I think the key to making this work in practice is to see the various steps in the RFC as 'strong recommendations' rather than rules, acknowledge that some project groups will be very different for one reason or another, and accept variations in the process.